types/hellosign-sdk/index.d.ts

```diff
@@ -686,6 +653,33 @@ declare namespace HelloSign {
         update(clientId: string, options: ApiAppRequestOptions): Promise<ApiAppResponse>;
         delete(clientId: string): Promise<BaseResponse>;
     }
+
+    class HelloSign {
+        constructor(options: HelloSignOptions);
+        account: AccountModule;
+        signatureRequest: SignatureRequestModule;
+        embedded: EmbeddedModule;
+        oauth: OAuthModule;
+        team: TeamModule;
+        template: TemplateModule;
+        reports: ReportsModule;
+        unclaimedDraft: UnclaimedDraftModule;
+        apiApp: ApiAppModule;
+        setHost: (host: string, port: string, protocol: string) => void;
+        setProtocol: (protocol: string) => void;
+        setPort: (port: string) => void;
+        setClientId: (clientId: string) => void;
+        setClientSecret: (clientSecret: string) => void;
+        setOauthToken: (oauthToken: string) => void;
+        setApiKey: (key: string) => void;
+        setUserPassAuth: (username: string, password: string) => void;
+        setTimeout: (timeout: number) => void;
+        _setApiField: (key: string, value: any) => void;
+        getApiField: (key: string) => any;
+        isDev: () => boolean;
+    }
 }
 
-export = HelloSign;
+declare function hellosign(options: HelloSign.HelloSignOptions): HelloSign.HelloSign;
```

Comment:
In the source code, I didn't see any function named `hellosign`. The exported object is a constructor function named `HelloSign`. There isn't an export named `default` either. I think these last 3 lines should stay the way they are.